PyU4V/tests/ci_tests/test_pyu4v_ci_provisioning.py

```diff
@@ -568,7 +569,22 @@ def test_modify_initiator_set_fcid(self):
 
     def test_modify_initiator_change_initiator_flags(self):
         """Test modify_initiator change initiator flags."""
-        initiator = self.provisioning.get_in_use_initiator()
+        initiator_name = '1%015x' % random.randrange(16**15)
+        host = self.create_host(initiator_list=[initiator_name])
+        port_group_name, _ = self.create_port_group()
+        masking_view_name = self.generate_name('masking_view')
+        storage_group_name = self.create_empty_storage_group()
+        self.provisioning.add_new_volume_to_storage_group(
+            storage_group_name, 1, 1, 'GB')
+        device_id = (
+            self.provisioning.get_volumes_from_storage_group(
+                storage_group_name)[0])
+        self.addCleanup(self.delete_volume, storage_group_name, device_id)
+        self.provisioning.create_masking_view_existing_components(
+            port_group_name, masking_view_name, storage_group_name, host)
+        self.addCleanup(self.delete_masking_view, masking_view_name)
+        initiator_list = self.provisioning.get_initiator_list()
+        initiator = [x for x in initiator_list if initiator_name in x][0]
```

Comment:
This is potentially dangerous? If initiator_name is not matched you will try to get the first element in an empty list

Reply:
@odonos12 can you re-examine this change

Reply:
At this point we've already created a host & masking view with an initiator (initiator_name) set, is there still a possibility that get_initiator_list() would still return a list that doesn't contain our initiator?

Reply:
You could assert the list is populated then extract the initiator name afterwards, but if you are confident that we wont ever run into that scenario where the initiator name isn't in the list then it will be fine as is

Reply:
@odonos12  are you ok to mark this as resolved?